src/examples/forms/range/RangeDisabled.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Field, Label, Range } from '@zendeskgarden/react-forms';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => {
+  return (
```

Comment:
Implicit returns are used elsewhere in this repo and I think that's the preferred style if functions can be written to implicitly return values.

Reply:
Yes, implicit returns throughout when possible, please.